org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
```

Comment:
Makes sense. Is it possible to make a separate PR for any schema changes though, so it's really obvious or could you update the name of this PR to "Modified Schema and Added Nest Thermostat"